### ISSUE TITLE
Unable to parse one-liner with background-size attribute

### ISSUE THREAD

**Issue by daviestar** [yuezk/postcss-filter-gradient#8]
Hello,

Thanks for this plugin.

It failed to parse this CSS line:

``` sass
background: linear-gradient(#626262, #626262 80%, transparent 80%, transparent) 0 bottom / 100% 12px
```

Error:

```
Error: Unable to parse color from string "12p"
```

**Comment by yuezk** [comment]
Thanks for your feedback. So, how should we handle this? Just omit the `background-size` value?

**Comment by daviestar** [comment]
Only parse within the parenthesis.  

I discovered this didn't work:

``` sass
background: linear-gradient(#626262, #626262 80%, transparent 80%, transparent) 0 bottom
  size: 100% 12px

// Unable to parse color from string "botto"
```

but this did:

``` sass
background: linear-gradient(#626262, #626262 80%, transparent 80%, transparent)
  position: 0 bottom
  size:  100% 12px
```

Or just put in your documentation that the gradient has to be on it's own line :)

Thanks

**Comment by daviestar** [comment]
I had a look at the code, and the problem lies with this line: https://github.com/yuezk/postcss-filter-gradient/blob/master/index.js#L206

CSS gradients behave like the `background-image` property.  You could write it like this:

``` css
div {
  background-image: url(...)
  background-position: top left
  background-size: 50px 100%
  background-repeat: no-repeat
}
// as one-liner
div {
  background: url(...) top left / 50px 100% no-repeat
}

// gradient
div {
  background-image: linear-gradient(...)
  background-position: top left
  background-size: 50px 100%
  background-repeat: no-repeat
}
// as one-liner
div {
  background: linear-gradient(...) top left / 50px 100% no-repeat
}
```

My line above is actually a repeating gradient which can't be replicated in IE filter, but the plugin should not error as this is valid CSS.  Have a look at [this SO question](http://stackoverflow.com/questions/546433/regular-expression-to-match-outer-brackets) for better ways to make sure you only parse what's inside `linear-gradient(...)`.  

Cheers.

**Comment by yuezk** [comment]
OK, thanks, I will fix this bug soon.

**Comment by yuezk** [comment]
Fixed in v0.2.4.

**Comment by daviestar** [comment]
Thanks, working for me :+1:
